app/routes/overview/components/Overview.css

```diff
@@ -52,11 +33,19 @@
   transition: color 0.2s;
   font-size: 1rem;
   font-weight: 600;
-  color: rgba(var(--rgb-min), var(--rgb-min), var(--rgb-min), 80%);
+  color: var(--color-dark-gray-5);
 }
 
-.item:hover .itemInfo {
-  color: black;
+.type {
+  margin-right: 4px;
+}
+
+.tagline {
+  display: inline-block;
+  white-space: nowrap;
+  transform: translateY(-7.5%);
```

Comment:
I take no criticism on this. It's the only way in which we keep the `vertical-align: middel`, which is needed for this component to work on both the pinned article and the event items below 🙃 FlexBox? Nope, that would ruin the centered alignment of the text on the event items below. Should probably be separated, but that's out of scope of what I'm trying to accomplish here. It still looks good on mobile, I checked.